tests/core/interfaces-gpio-chardev/task.yaml

```
@@ -0,0 +1,134 @@
+summary: Ensure that the GPIO chardev interface works.
+
+details: |
+    The gpio-chardev interface allows read/write access to specified GPIO chip lines.
+
+# Kernels on Ubuntu Core <24 does not contain the gpio-sim module.
+systems: [ubuntu-core-24]
```

maybe we can make this future proof and exclude core18/core20/core22, so that the test will be picked up for uc26 when the time comes
```suggestion
systems: [-ubuntu-core-18-*, -ubuntu-core-20-*, -ubuntu-core-22-*]
```

---

Good point, Thank you!